src/modules/commander/commander.cpp

```diff
@@ -1198,6 +1198,7 @@ int commander_thread_main(int argc, char *argv[])
 	bool was_armed = false;
 
 	bool startup_in_hil = false;
+	bool auto_disarm_enable = false; //the flag indicate enable auto disarm when the vehicle state from inair to land
```

Comment:
how? could you please fix the bug if you are free? and I will close the pull request